Enterprise-CMCS/macpro-mdct-mfp
# [MFP] WP PRA Disclosure

### Description



### Related ticket(s)


CMDCT-2947

---
### How to test

- Create a work plan
- Go to the general information page
- See the PRA disclosure
- Confirm you can only see it on this page and no others

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
